api/api_utils.py

```diff
@@ -19,8 +19,12 @@ def create_response(orig_response):
 
 
 def list_teams(dbsession=session):
-    teams = dbsession.query(Team).all()
```

Comment:
Previously I had to call `expunge(obj)` on objects that are generated with the query command as they were linked with the database i.e. attached to the session.